### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Dewlar**
+ add header component
+ add footer component
+ add home page:
    + hero(promo) section
    + collection preview section
    + story section
    + policy section
+ add footer component
+ add 404 page

### CODE REVIEW

File: src/app/components/header/header.tsx [hunk 1]
```diff
@@ -1,7 +1,100 @@
...
+                <Link to="/">
+                  <span className="sr-only">Green shop</span>
+                  <img className="h-8 w-auto" src="./assets/logo/logo.png" alt="logo" />
+                </Link>
+              </div>
+
+              <div className="hidden h-full lg:flex">
+                <div className="flex h-full justify-center items-center space-x-8">
+                  {mocks.navigation.pages.map((item) => {
+                    console.log(item);
```
Comment (letanatol) [inline src/app/components/header/header.tsx:29]: Console log must remove!

File: src/app/components/header/header.tsx [hunk 2]
```diff
@@ -1,7 +1,100 @@
...
+                    return (
+                      <Link key={item.name} to={item.href} className="text-gray-500 hover:text-gray-600">
+                        {item.name}
+                      </Link>
+                    );
+                  })}
+                </div>
+                {/* Flyout menus */}
```
Comment (letanatol) [inline src/app/components/header/header.tsx:37]:
```suggestion
```

**Review by letanatol — APPROVED** [review]
Very, very good!

File: src/app/components/header/header.tsx (hunk 1 above)
Comment (Dewlar) [reply src/app/components/header/header.tsx:29]: okay

**Review by letanatol — APPROVED** [review]

**Review by letanatol — APPROVED** [review]
Very, very good!

**Pull request merged**